openmls/src/framing/validation.rs

```diff
@@ -0,0 +1,266 @@
+//! # Validation steps for incoming messages
+//! ```text
+//! MlsMessageIn (exposes: wire format, group, epoch)
+//! |
+//! V
+//! parse_message()
+//! |
+//! V
+//! DecryptedMessage
```

Comment:
This state is not really visible to the outside, though, is it? Or at least it's not the return type of `parse_message`.